doc/examples/test-examples.js

```diff
@@ -0,0 +1,25 @@
+const { readdirSync, statSync } = require('fs');
+const { join } = require('path');
+const execa = require('execa');
+const exampleDirs = readdirSync(__dirname)
+	.map(dir => join(__dirname, dir))
+	.filter(dir => statSync(dir).isDirectory());
+
+async function runner(dir) {
+	const config = { cwd: dir, stdio: 'inherit' };
+
+	let install = await execa.shell('npm install', config);
+	let test = await execa.shell('npm test', config);
+
+	return Promise.all([install, test]);
```

Comment:
Since the `install` and `test` promises have already resolved, we don't need the `Promise.all` here 😄

Reply:
Yeah, I realized that after I pushed. Sorry about that!